proto/depot/cli/v1/build.proto

```diff
@@ -274,6 +275,7 @@ message GetPullInfoResponse {
   string username = 2;
   string password = 3;
   repeated BuildOptions options = 4;
+  bool saveForLoad = 5;
```

Comment:
We've been using the [Buf](https://marketplace.visualstudio.com/items?itemName=bufbuild.vscode-buf) vscode to do linting and formatting for the protos:

```suggestion
  bool save_for_load = 5;
```